src/ckb-daemon/usb.h

```diff
@@ -39,6 +39,7 @@
 #define P_K95_PLATINUM  0x1b2d
 #define P_K95_PLATINUM_STR "1b2d"
 #define IS_K95(kb)      ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95 || (kb)->product == P_K95_NRGB || (kb)->product == P_K95_PLATINUM))
+#define IS_K95P(kb)     ((kb)->vendor == V_CORSAIR && ((kb)->product == P_K95_PLATINUM))
```

Comment:
Yes, it's still a K95, the P is used to only apply the workarounds for the specific device. I do think it should be called IS_PLATINUM though, as I sense more devices coming up.